internal/sim/executor.go

```diff
@@ -286,6 +287,8 @@ func (e *executor) reset(workload Workload, fakes map[reflect.Type]any, ops []*o
 	e.nextTraceID = 1
 	e.nextSpanID = 1
 
+	id := uuid.New().String()[:8]
```

Comment:
I'm not sure if it's 100% necessary, but we might want the deployment id to be deterministic. What do you think about using a function like the following to generate the id? We can call it with `e.rand` as an argument.

```go
func newUUID(r *rand.Rand) (uuid.UUID, error) {
    var bytes [16]byte
    r.Read(bytes[:])
    return uuid.FromBytes(bytes[:])
}
```

Reply:
Thanks, done!